Description: fix regexp match against `file /proc/$$/exe` for -fPIE bash
 Now that bash is built with PIE enabled, keyutils' check for endianness
 fails because file no longer returns "executable", but instead returns
 "shared object".  Update our regexps to be portable.
Author: Steve Langasek <steve.langasek@ubuntu.com>
Last-Modified: 2018-04-17

Index: keyutils-1.5.9/tests/toolbox.inc.sh
===================================================================
--- keyutils-1.5.9.orig/tests/toolbox.inc.sh
+++ keyutils-1.5.9/tests/toolbox.inc.sh
@@ -19,10 +19,10 @@
 echo === $OUTPUTFILE ===
 
 endian=`file -L /proc/$$/exe`
-if expr "$endian" : '.* MSB executable.*' >&/dev/null
+if expr "$endian" : '.* MSB \(executable\|shared object\).*' >&/dev/null
 then
     endian=BE
-elif expr "$endian" : '.* LSB executable.*' >&/dev/null
+elif expr "$endian" : '.* LSB \(executable\|shared object\).*' >&/dev/null
 then
     endian=LE
 else
